R/scale.R

```diff
@@ -110,6 +110,20 @@ prep.step_scale <- function(x, training, info = NULL, ...) {
 
   sds <- sds * x$factor
 
+  which_sd <- which(sds ==0)
+  if (length(which_sd) > 0) {
+    with_na <- names(which_sd)
+    with_na_str <- paste(paste0("`", with_na, "`"), collapse = ", ")
```

Comment:
recipes already imports [glue](https://glue.tidyverse.org/) so let's use that.

Reply:
Ok will change it.

Reply:
Done